containers/ddev-webserver/ddev-webserver-base-files/etc/php/8.0/mods-available/assert.ini

```diff
@@ -0,0 +1,2 @@
+[Assertion]
```

Comment:
This looks pretty reasonable to me. I'd say just add a comment to these assert.ini files so that people don't wonder where they came from.

Reply:
that is the default module/name. See https://github.com/php/php-src/blob/master/php.ini-production#L1569